docs/content/add-plugins/s3.md

```diff
@@ -150,7 +150,7 @@ You can also check that the image was uploaded on the S3 bucket’s page.
 
 ## Next.js Storefront Configuration
 
-If you’re using a [Next.js](../starters/nextjs-medusa-starter.md storefront, you need to add an additional configuration that adds the S3 bucket domain name into the configured images’ domain names. This is because all URLs of product images will be from the S3 bucket.
+If you’re using a [Next.js](https://docs.medusajs.com/starters/nextjs-medusa-starter/) storefront, you need to add an additional configuration that adds the S3 bucket domain name into the configured images’ domain names. This is because all URLs of product images will be from the S3 bucket.
```

Comment:
The link should be a markdown link as showcased in the [Docusaurus documentation](https://docusaurus.io/docs/markdown-features/links). So for example, it could be something like `../starters/nextjs-medusa-starter.md` (not sure if this is the correct path so please check)